portal_customer_wallet/tests/common.py

```diff
@@ -0,0 +1,134 @@
+                        },
+                    )
+                ],
+                "outbound_payment_method_line_ids": [
+                    (
+                        0,
+                        False,
+                        {
+                            "payment_method_id": cls.env.ref(
+                                "account.account_payment_method_manual_out"
+                            ).id,
+                            "name": "Manual",
+                            "payment_account_id": cls.customer_wallet_account.id,
+                        },
+                    )
+                ],
+            }
+        )
+        cls.cash_account = cls.env["account.account"].create(
+            {
+                "name": "Test Cash Account",
+                "code": "654321",
+                "account_type": "asset_cash",
+            }
+        )
+        cls.cash_journal = cls.env["account.journal"].create(
+            {
+                "name": "Test Cash Journal",
+                "code": "TSTCASH",
+                "type": "cash",
+                "inbound_payment_method_line_ids": [
+                    (
+                        0,
+                        False,
+                        {
+                            "payment_method_id": cls.env.ref(
+                                "account.account_payment_method_manual_in"
+                            ).id,
+                            "name": "Manual",
+                            "payment_account_id": cls.cash_account.id,
+                        },
+                    )
+                ],
+                "outbound_payment_method_line_ids": [
+                    (
+                        0,
+                        False,
+                        {
+                            "payment_method_id": cls.env.ref(
+                                "account.account_payment_method_manual_out"
+                            ).id,
+                            "name": "Manual",
+                            "payment_account_id": cls.cash_account.id,
+                        },
+                    )
+                ],
+            }
+        )
+
+    def _create_move(self, debit=0, credit=0, date=None, partner=None):
+        if partner is None:
+            partner = self.partner_id
+        if date is None:
+            date = datetime.date.today()
+
+        self.env["account.move"].create(
+            {
+                "journal_id": self.customer_wallet_journal.id,
+                "date": date,
+                "line_ids": [
+                    (
+                        0,
+                        0,
+                        {
+                            "debit": debit,
+                            "credit": credit,
+                            "partner_id": partner.id,
+                            "account_id": self.customer_wallet_account.id,
+                        },
+                    ),
```

Comment:
please use the [new syntax](https://www.odoo.com/documentation/15.0/developer/reference/backend/orm.html#odoo.fields.Command) for 15.0+: `fields.Command.create(vals)`